src/components/index.ts

```diff
@@ -1 +1,3 @@
 export { Typography } from './Typography'
+export { Sidebar } from './Sidebar'
+export { SidebarItem } from './SidebarItem'
```

Comment:
If `SidebarItem` is only meant to be used as a `Sidebar` internal (maybe it's not?), I wouldn't expose it on the same level as `Sidebar`

Reply:
What do you think of a `components` folder inside the `Sidebar` folder with an `index.tsx` file that exposes all the internal components ? It would prevent to expose internal components at the app level and would give an import statement such as `import { SidebarItem } from './components'`

Reply:
Perfect for me 👌